index.html

```diff
@@ -12,12 +12,37 @@ <h2>Groceries list</h2>
     <h3>Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis voluptatem tempora quisquam maxime libero ipsam rerum sit, ducimus laboriosam saepe veniam neque adipisci praesentium labore. Esse cupiditate perspiciatis ea at.
     Totam, nihil! Omnis eligendi quis enim quidem maxime vitae quam ipsum accusantium esse eos sequi tenetur id repellat quisquam dicta asperiores repellendus, at suscipit sunt ex! Molestias, architecto dignissimos? Sapiente?</h3>
     <h4>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Velit, similique accusamus. Cum tempore corporis quibusdam perspiciatis magni non illo commodi porro, voluptatum, dolores minima voluptate recusandae est sint veritatis ullam?</h4>
-    <h2>Items to buy:</h2>
+    <h3>Items to buy:</h3>
     <ol>
         <li>Bread</li>
         <li>Salt</li>
         <li>Sugar</li>
         <li>Paper</li>
         <li>Pork</li>
+<h2>Registration form</h2>
+
+<label>
+          <p>Address info</p>
+          <hr>
```

Comment:
In order to achieve the same result as you see at the screenshot, you can use the `fieldset` tag - https://doka.guide/html/fieldset/